src/Component/legacy/src/Exception/StorageUnavailableException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Exception;
+
+class StorageUnavailableException extends \RuntimeException
```

Comment:
Exceptions are modernized at new non-component namespace, please apply this change.

```suggestion
use Sylius\Resource\Exception\RuntimeException;

class StorageUnavailableException extends RuntimeException
```

![image](https://github.com/Sylius/SyliusResourceBundle/assets/870747/7c842248-7be5-4ad1-b464-a83d9e2f7db3)